recipes/libunrar/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "libunrar" %}
+{% set version = "6.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://www.rarlab.com/rar/unrarsrc-{{ version }}.tar.gz
+  sha256: 120936e41f826cd63d77a580aeea64c1b79fd3e2434f58ce8184e783b51e5b01
+  patches:
+    - 0001-REMOVE-HARDCODED-MAKEFILE-ENV.patch  # [unix]
+    - 0002-REMOVE-INSTALL-OPTION.patch  # [osx]
+    - 0003-CHANGE-DYLIB-EXTENSION.patch  # [osx]
+
+build:
```

Comment:
How is the ABI stability of this lib? It will need a matching run_exports

Reply:
The ABI stability is really, really bad and changes on every minor version. Not even its own utility can link to the lib. I think it's only meant to be used statically.

That said, this is the official implementation of RAR unpacking and the community alternatives are not able to achieve feature parity with the newest non-free format.